superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -516,6 +516,15 @@ def _get_ros_version(distro):
         return 2 if distro not in distros \
             else int(distros[distro]['distribution_type'][len('ros'):])
 
+    @staticmethod
+    def _get_superflore_version():
```

Comment:
@andre-rosa I think that this function would better find a home in `utils.py`, since it's not open embedded specific.

Reply:
Yeah, I was thinking about that too. Even though it has just one user now it's better to land it directly into ```utils.py```.

Reply:
Done